dev-tools/jenkins_ci.ps1

```diff
@@ -36,13 +36,14 @@ exec { go get -u github.com/jstemmer/go-junit-report }
 echo "Building $env:beat"
 exec { go build } "Build FAILURE"
 
+# always build the libbeat fields
+cp ..\libbeat\_meta\fields.common.yml ..\libbeat\_meta\fields.generated.yml
```

Comment:
As soon as https://github.com/elastic/beats/pull/6911 is in this should become much simpler.